File: AreaTriangulo.scala
import scala.io.StdIn.readLine
object AreaTSC {
    def main(args: Array[String]) = {
      //Declaracion de variables y otros
        println("Ejercicio 01:Area triangulo")
      //Datos de entrada
      println("Ingrese Base:")
      val b = readLine().toInt
      println("Ingrese Altura:")
      val h = readLine().toInt
      //proceso
      val area=(b*h)/2
      //Datos de salida
      println("El area del triangulo es:"+area)
    }
}

File: distincg-lenguaje/AreaTriangulo.dart
import 'dart:io';
void main(){
  //Defenir Variables y otros
  print('Ejercicio 01:Area triangulo');
  //Datos de Entrada
  print("Ingrese Base: ");
  var b=int.parse(stdin.readLineSync());
  print("Ingrese Altura: ");
  var h=int.parse(stdin.readLineSync());
  //Proceso
  var area=(b*h)/2;
  //Datos de salida
  print("El Area de Triangulo es: $area");
}